tests/frontend/test_about.py

```diff
@@ -14,9 +14,14 @@
 """
 Unit tests for top-level Strawberry Fields functions.
 """
+import pytest
```

Comment:
Why did these changes effect this test?

Reply:
I realised this test wasn't being marked as a frontend test, so it was running numerous times.